src/remix.ts

```diff
@@ -0,0 +1,16 @@
+import { redirect, json } from "@remix-run/server-runtime";
+import { Authenticator, PublicAuthenticatorOptions } from "./authenticator";
+
+export class RemixAuthenticator<TUser, TFlash, TContext> extends Authenticator<
+  TUser,
+  TFlash,
+  TContext
+> {
+  constructor(options: PublicAuthenticatorOptions<TUser, TFlash, TContext>) {
+    super({
+      ...options,
+      json,
+      redirect,
+    });
+  }
+}
```

Comment:
I also imagine that this would need to stay in the "remix-auth" package while most everything else would need to go into some core package.